opencga-app/app/scripts/azure/arm/mongodb/postdeploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#this script is wrapped into the mongodb template in base64
+#just here for reference
+
+echo $APP_DNS_NAME
+echo $MONGODB_USERNAME
+echo $MONGODB_PASSWORD
+sudo apt-get update
```

Comment:
I don't think sudo is required as the Azure Script extension is runs as admin

Reply:
Removed sudo in https://github.com/opencb/opencga/pull/975/commits/1ec41d9d76237c8182b43e1b401c073730cfff12